packages/mui-codemod/src/deprecations/speed-dial-props/speed-dial-props.js

```diff
@@ -0,0 +1,20 @@
+import movePropIntoSlots from '../utils/movePropIntoSlots';
+
+/**
+ * @param {import('jscodeshift').FileInfo} file
+ * @param {import('jscodeshift').API} api
+ */
+export default function transformer(file, api, options) {
+  const j = api.jscodeshift;
+  const root = j(file.source);
+  const printOptions = options.printOptions;
+
+  movePropIntoSlots(j, {
+    root,
+    componentName: 'SpeedDial',
+    propName: 'TransitionComponent',
+    slotName: 'transition',
+  });
+
```

Comment:
We should add `movePropIntoSlotProps` for `TransitionProps`, for example: https://github.com/mui/material-ui/blob/48d033831d2d837514702abb83467e1d65034257/packages/mui-codemod/src/deprecations/accordion-props/accordion-props.js#L20